doc/standards/testStrategy/GeneralGuidelines.md

```diff
@@ -0,0 +1,68 @@
+# General guidelines
+
+## FIRST principles
+
+- Fast : Could be run anytime during the development phase
+- Independant/Isolated : Test should also follow 3A (Arrange, Act, Assert, ou Given-When-Then) and test one functionality
+- Repeatable : Each test sould give the same result each time it runs
+- Self-validating : The output of a test should be "yes" or "no", and not some values to check
+- Thorough : Tests must cover all possibilities and edge cases
+- Timely : Test should be written before their feature (TDD)
+
+---
+
+## When to use .web and .native?
+
+use `MyComponent.native.test.tsx` if
```

Comment:
à la première lecture, j'ai cru que c'était "if … and … and …", alors que c'est plutôt "if … or … or …" :/ 
Peut-être dire "either if" ?
(pareil pour les cas suivants)